pkg/smbios/info_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2016-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package smbios
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestGetBIOSInfo(t *testing.T) {
+	testutil.SkipIfNotRoot(t)
+
+	if runtime.GOARCH != "amd64" {
```

Comment:
just use build constraints or, better, change the file name. 

I assume this only works on linux and amd64?